formatting/action.yml

```diff
@@ -65,19 +65,29 @@ runs:
         echo -e "::group::${{ env.bashInfo }} ${{ env.stepName }} ${{ env.bashEnd }}"
 
         wget https://github.com/uncrustify/uncrustify/archive/refs/tags/uncrustify-0.69.0.tar.gz -O uncrustify-0.69.0.tar.gz
-        tar xzvf uncrustify-0.69.0.tar.gz && cd uncrustify-uncrustify-0.69.0
 
-        cmake -S . -B build -DCMAKE_BUILD_TYPE=Release
-        cmake --build build -- -j$(nproc) -s
+        EXPECTED_SHA256="33bd97a07f7c4bd114874f73171aca220bf05c17108f8505a117b97374a347b6"
```

Comment:
Nit: I wonder if we call this information more dynamically from uncrustify. Not blocking as I'd still like to get this check in ASAP.